### ISSUE TITLE
Error during Login

### ISSUE THREAD

**Issue by signalpoint** [signalpoint/force_authentication#1]
Issue reported by @amitsedai.

In the user login page after providing the login user name and password, it is not redirected to front page.

On clicking the Login button again, an error pops up:
error: (Not Acceptable: Already logged in as username.)["Already logged in as username."][undefined]

Also, the front page of the app can be shown for anonymous users. Based on whether a previous page has been set or not, redirecting anonymous users to login page can be worked upon.

What about reloading the front page and providing that to anonymous users instead of login?

Thanks.

**Comment by signalpoint** [comment]
@amitsedai - if you download the tagged release this bug will go away:

https://github.com/signalpoint/force_authentication/releases

The current 7.x-1.x dev version of this module is dependent on this branch of DrupalGap, which hasn't been merged into the main branch yet:

https://github.com/signalpoint/DrupalGap/tree/7.x-1.x-alpha-service-hooks

Once that branch is merged in, this module will have a new tagged release.

**Comment by amitsedai** [comment]
Thanks.

I enabled the module with the latest version and the error message didn't appear during login. However, after clicking on logout none of the links actually worked that appeared in the page. I got the following error message at the console:
TypeError: Result of expression 'i.data("mobile-page")' [undefined] is not an object. at file:///android_asset/www/jquery.mobile-1.3.1.min.js:3

This error message appears even without this module enabled after logout button is clicked.
